android/src/main/java/com/rnldk/RnLdkModule.kt

```diff
@@ -262,7 +269,7 @@ class RnLdkModule(private val reactContext: ReactApplicationContext) : ReactCont
           fee_estimator,
           chain_monitor,
           tx_filter,
-          router,
+          router!!.write(),
```

Comment:
@TheBlueMatt wtf?

Reply:
especially in below  call its still router, not bytes

Reply:
Right, well the intent here is to read it from disk, which should let you drop a bunch of the code around `initialize graph sync`